.github/workflows/android.yml

```
@@ -0,0 +1,72 @@
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        variant: [saksham_staging]  # Add other variants as needed
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up JDK
+        uses: actions/setup-java@v2
```

actionlint (1.7.4)

the runner of "actions/setup-java@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)